engine/src/main/scala/cromwell/services/EngineMetadataServiceActor.scala

```diff
@@ -23,6 +45,34 @@ case class EngineMetadataServiceActor(serviceConfig: Config, globalConfig: Confi
     }
   }
 
+  private def queryStatusAndRespond(id: WorkflowId): Unit = {
+    val sndr = sender()
+    dataAccess.getWorkflowStatus(id) onComplete {
```

Comment:
That was bad wording on my part, but your last sentence answered my question anyways :)